src/pages/terms-and-conditions/_business-grid.tsx

```diff
@@ -62,11 +62,11 @@ const PartnersGuidePdf = () => {
 }
 
 const BusinessGrid = () => {
-    const general_terms_url = `/tnc/business-partners-general-terms-es.pdf`
-    const affiliate_brokers_url = `/tnc/business-partners-affiliates-and-introducing-brokers-es.pdf`
-    const payment_agents_url = `/tnc/business-partners-payment-agents-es.pdf`
-    const api_user_url = `/tnc/business-partners-api-user-es.pdf`
-    const bug_bounty_url = `/tnc/business-partners-bug-bounty-es.pdf`
+    const general_terms_url = `/tnc/[ES] business-partners-general-terms-of-use - R23_12_1.pdf`
```

Comment:
Hi @agrim-deriv ,
I don’t think its good to have spaces in file name. 

@habib-deriv, @hasan-deriv  please correct me if I am wrong.

Reply:
Made the changes @fasihali-deriv